packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/double-close-div/actual.html

```diff
@@ -0,0 +1,3 @@
+<template>
+    <div></div></div>
+</template>
```

Comment:
Confirmed that this `</div></div>` was also supported (no warnings, no errors) in 234 (v2.2.12).